commands/customObject/schema/delete.ts

```diff
@@ -10,17 +14,37 @@ const { logError } = require('../../../lib/errorHandlers');
 
 const i18nKey = 'commands.customObject.subcommands.schema.subcommands.delete';
 
-exports.command = 'delete <name>';
+exports.command = 'delete [name]';
 exports.describe = i18n(`${i18nKey}.describe`);
 
 exports.handler = async options => {
-  const { name, derivedAccountId } = options;
+  const { name: providedName, force, derivedAccountId } = options;
 
   await loadAndValidateOptions(options);
 
   trackCommandUsage('custom-object-schema-delete', null, derivedAccountId);
 
+  let name;
   try {
+    const {
+      data: { results },
+    } = await fetchObjectSchemas(derivedAccountId);
+    const schemaNames = results?.map(({ name: schemaName }) => schemaName);
+    name =
+      providedName ||
+      (await listPrompt(i18n(`${i18nKey}.selectSchema`), {
+        choices: schemaNames,
+      }));
+
+    const shouldDelete =
+      force ||
+      (await confirmPrompt(i18n(`${i18nKey}.confirmDelete`, { name })));
+
+    if (!shouldDelete) {
+      logger.success(i18n(`${i18nKey}.deleteCancelled`, { name }));
```

Comment:
Feel like this might make more sense as a `log` or `info`

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1261/commits/55b705e95243103a419bf87e4dafa3a0f3d64080